app/components/AndroidListView.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  ListView,
+  StyleSheet,
+  Text,
+  View,
+  TouchableHighlight
+} from 'react-native';
+
+import mockData from '../mockData.json' // TEMP
+import { ChallengeDetailRoute } from '../routes/defaultRoutes'
+
+export default class AndroidListView extends Component {
+  constructor(props) {
+    super(props);
+    this.goToChallengeDetailScene = this.goToChallengeDetailScene.bind(this);
+    this.dataSource = new ListView.DataSource({
+      rowHasChanged: (r1, r2) => r1 !== r2
+    })
+    this.state = {
+      mockData: mockData.challenges
+    };
+  }
+
+  goToChallengeDetailScene() {
+    var selectedChallenge = mockData.challenges[0]; // TEMP CHALLENGE MOCK
+
+    // build route
+    var route = ChallengeDetailRoute;
```

Comment:
We the only property being changed is the title (if one exists). After that, there are only props being _added_. While it's true you can create your own route, that's not what we want to do and this the preferred method. First, if you are creating your own route, you will have to import the component to be rendered on that route in order to add it to your route object. So we aren't saving anything with imports, it's just whether your import the defaultRoute, or the component and then build the object and put that component inside of it. Think of defaultRoutes like templates, they contain the most common values that will be used in 90% of the use cases, but are still modifiable if specific things need to change. Second, the defaultRoutes gives us a single source of truth, rather than people people configuring scenes to render components in any way they want. It's important to note that most of these benefits aren't very noticeable at the moment since there is not a lot in the defaultRoutes. That will not always be the case. There are still default values I will need to add in when implementing the custom animations and transitions. Rather than copy and paste that into all the places where those routes are being used, I can just added them to the defaultRoutes and everyone will get those values. Along those same lines, if the parent component in a scene is ever changed, there is only one place where that change needs to take place. It also provides a single places to look to see what are all of the scenes we have defined in our app, similiar to a registerRoutes file in an express server.